code-challenges.test.js

```diff
@@ -50,3 +130,40 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+// Input: an array of numbers
+// Output: whether or not the array is a "full house"
+
+// Psuedo Code: create an higher order funtion  that first creates an empty object called counts to store the number of occurrences of each number in the input array.then loops throuhg the input array and increments the count for each number in the counts object.After that the  function checks if there are three of one number and two of another by looping through the values of the counts object and setting  Let hasThree and hasTwo to true if the  count is found. Finally, the function returns true if both lets are set to true, showing  full house, and false otherwise.
+
+
+
+
+
+
+function fullHouse(numbers) {
+ // creates an empty object called counts
```

Comment:
This is a very interesting approach. Is there a reason you decided to write this function in this particular way? I would also include the way I intend to loop through the array in pseudocode to make it more descriptive.